kube/objects.go

```diff
@@ -182,7 +190,16 @@ func (c *Client) FetchObjects(ctx context.Context, filter ObjectFilter) (*Object
 		err = annotateFetchError("CronJobs", err)
 		return
 	})
-
+	g.Go(func() (err error) {
+		objects.VolumeSnapshotsV1, err = csiClient.VolumeSnapshots(corev1.NamespaceAll).List(ctx, filter.NamespaceOptions(opts))
```

Comment:
Note that VolumeSnapshotContent objects also need to be checked for the respective annotation [according to the example in the upstream docs](https://github.com/kubernetes-csi/external-snapshotter/tree/eb447544376eaf7115cfe70bd143a40e7718262a#validating-webhook).